ground/src/main/java/com/google/android/ground/model/geometry/Point.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.geometry
+
+import com.google.common.collect.ImmutableList
+
+/** An individual point. */
+data class Point(override val coordinate: Coordinate) : Geometry {
+    override val coordinates: ImmutableList<Coordinate> = ImmutableList.of(coordinate)
```

Comment:
Consider not using `coordinates` in all Geometry implementations, instead modeling the semantics of geometry try, as we do in https://github.com/google/ground-platform/tree/master/web/src/app/shared/models/geometry

Reply:
I think we might be trying to force polymorphism by adding a common attribute to all geometries. I know the JTS libraries do this, but a) I haven't seen a need for these in our code yet, and b) it's unclear how those would be useful at all.

Reply:
Ah yeah, you're right, I was looking at the converter, which I believe we're keeping as JTS for JSON reads, right? ok sg, I can drop these.

Reply:
The converters read/write to GeoJSON, which is what JTS does as well. In that case, `coordinates` has a different meaning and structure for each geometry type. Ideally, we wouldn't leak the serialized representation into our domain model; those details are currently encapsulated in `GeometryConverter`, so no need to add them here as well.